### PR TITLE
[ADD] inventory: stock report

### PR DESCRIPTION
**Pull request by Felicious**
## Summary
New doc explaining what the Stock report displays.
- First of a series of Inventory reporting docs. 
- Includes section explaining the page and its fields
- Description of each filter option available to customize report
- pencil icon!

Note to reviewers: As this is the first of all Inventory reporting docs, let me know if you agree/disagree on where these docs are located in the toc tree!

[Task](https://www.odoo.com/web#id=3920184&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form)

### CODE REVIEW

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/stock.rst
Comment (larm-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/stock.rst:10]:
```suggestion
.. note::
   The reporting feature is only accessible to users with :doc:`admin access
   <../../../../general/users/access_rights>`.
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/stock.rst
Comment (larm-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/stock.rst:22]:
```suggestion
On the stock report, the left sidebar includes several groupings to narrow down what is being shown. The default groupigs are :guilabel:`Warehouses`,  which filters products by specific
warehouses, and :guilabel:`Category`, which shows products within a selected product category.
```

I would also add a note that the Wareouses only shows up if in a ulti=warehouse nevironment. In Runbot I only see the categories.

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/stock.rst
Comment (larm-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/stock.rst:27]: would a link to the documentation on valuation help? I know there is aFIFO, LIFO, AVCO, etc- and it cna affect the cost? Is that relevant to mention or link to?

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/stock.rst
Comment (larm-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/stock.rst:32]: I think I am confused- is the cost, the cost to purchase or manufacture? And then the value is different? I just think they can be explained a touch further since I am not 100% sure the difference between the two.

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/stock.rst
Comment (larm-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/stock.rst:38]:
```suggestion
- :guilabel:`Free to Use`: on-hand quantity that are **not** reserved for delivery or manufacturing
  orders, and are available to sell or use.
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/stock.rst
Comment (larm-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/stock.rst:39]: Is this based on the number of approved/confirmed PO's? I might expand this so we know where this number is coming from. I assume it's fro the PO- but is it nly ocunting confirmed, any on order, quuotes requested, etc?

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/stock.rst
Comment (larm-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/stock.rst:41]: again- expected ot leave on coonfirmes sales orders? I'd just be super specific.

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/stock.rst
Comment (larm-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/stock.rst:51]: in general- I noticed the buttons appear regardless if there is info to show or not. Example, when I clicked 'replenishment' it took me to a screen that said 'there are no re-ordering rules for this product'. So, when you click it, does it hsow the rule,s or does it actually reorder? I might oduble check for that, and let people know the buttons appear regardless if there is info?

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/stock.rst
Comment (larm-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/stock.rst:57]: you can also reference a generic reporting doc- I believe there is one that explains the search/filter/etc?

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/stock.rst
Comment (larm-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/stock.rst:72]: ooh i like the youtube link!! GOOD IDEA!!!

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/stock.rst
Comment (larm-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/stock.rst:75]: could possibly link to the product form part of a doc for adding a product?

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/stock.rst
Comment (larm-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/stock.rst:76]: also labeled as such on the product form?

**Review by larm-odoo — APPROVED** [review]
Hi @Felicious - great new doc! I like the idea for the structure, having reporting being a whole seciton with each report having it's own doc. I do think there's some room for expansion and to link to more docs/sections of docs. It references lots of types of things which maybe a user might need more info on. When these are all publishe,d this will be a great section! I am approving now since I think you can make these suggestions if you want- nothing is requied, just things I htink could make this a bit 'beefire' or add some more info. But again, they are not necessary, just suggesitons. lMK if you have any quesitons!

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/stock.rst
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/stock.rst:27]: Added the costing methods doc! thank you for the suggestion :smile:

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/stock.rst
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/stock.rst:32]: Added some more details! thank you (:

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/stock.rst
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/stock.rst:41]: Yes, thank you! Adding the clarification helps

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/stock.rst
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/stock.rst:51]: Suggestion implemented (:

**Review by StraubCreative — APPROVED** [review]
Hi @Felicious 

I think this is a good start to a series of reporting docs we can build out for Inventory and the other materials apps. Good UI and menu item coverage here, nice job 👍 

What you have here should get users started with understanding the UI right away, which is helpful! Later, for this particular dashboard, I believe there are some deep use cases involving specific filters, that we will either want to add to this doc to continue building it out, or can make new docs.

Couple quick comments for you below. Once those are addressed you can merge 🚀 
.....
@robodoo delegate=Felicious

**Comment by Felicious** [comment]
@robodoo r+

**Pull request closed without merging**
